activities.json

```diff
@@ -109,6 +109,17 @@
     "title": "Feature Policy",
     "url": "https://wicg.github.io/feature-policy"
   },
+  {
+    "ciuName": null,
+    "description": "An extension of the ES6 Script Modules system to include HTML Modules. These will allow web developers to package and access declarative content from script in a way that allows for good componentization and reusability, and integrates well into the existing ES6 Modules infrastructure.",
+    "mozBugUrl": null,
+    "mozPosition": "worth prototyping",
+    "mozPositionDetail": null,
+    "mozPositionIssue": 137,
+    "org": "W3C",
```

Comment:
I think the plan of record is a PR against the HTML Standard, so this should probably be WHATWG. We could also change it later though.

Reply:
I was going to... and then I realized that the system wasn't set up for "org": "WHATWG"... but we should probably fix that.